README.md

```diff
@@ -5,7 +5,7 @@
 
 ### behind the moar for performance ...
 
-JET.jl employs Julia's type inference for bug reports.
+JET.jl employs Julia's type inference to detect potential bugs.
```

Comment:
It's a bit confusing to have "moar for performance" in the title and then have the entire README focused on correctness (an awesome goal in its own right) rather than JET's performance-related runtime dispatch detection.

Reply:
That title (and the repository description) is what I wrote very randomly years ago, and now I'm feeling somewhat a bit embarrassed to have them lol